#ir message 
uint16[5] ir_analog
bool[5] ir_digital

#get the system time when passing sensor data
float32 new_t
